compiler/x/codegen/OMRInstOpCode.enum

```diff
@@ -26,6 +26,8 @@
 
 #include "compiler/codegen/OMRInstOpCode.enum"
 
+   ud2,
+   int3,
```

Comment:
Can you insert these into their somewhat alphabetical place amongst the other instructions?  I only just noticed that the list isn't already perfectly sorted (and we should change that at some point), but if we can still fit these in more or less the right place it make searching the opcodes by a human a lot easier.